pkg/vm/stack.go

```diff
@@ -117,7 +99,19 @@ func (e *Element) Bool() bool {
 // Bytes attempts to get the underlying value of the element as a byte array.
 // Will panic if the assertion failed which will be caught by the VM.
 func (e *Element) Bytes() []byte {
-	return e.value.Value().([]byte)
+	switch t := e.value.(type) {
```

Comment:
I believe these changes  should be in separate commit not in "add tests for CAT"